index.js

```diff
@@ -199,9 +199,23 @@ const captureWebsite = async (url, options) => {
 			timeout: timeoutInSeconds
 		});
 		screenshotOptions.clip = await page.$eval(options.element, getBoundingClientRect);
+		if (options.inset) {
+			const {inset} = options;
+			screenshotOptions.clip.width -= (inset.left + inset.right) || inset * 2;
+			screenshotOptions.clip.height -= (inset.top + inset.bottom) || inset * 2;
+			screenshotOptions.clip.x += inset.left || inset;
+			screenshotOptions.clip.y += inset.top || inset;
+		}
+
 		screenshotOptions.fullPage = false;
 	}
 
+	if (!options.element && options.inset && options.inset.top) {
+		await Promise.all([
```

Comment:
Using Promise.all here is weird as it's only one element.

Reply:
removed, it not needed. previously I was using evaluate function with async.